app/controllers/application_controller.rb

```diff
@@ -531,6 +531,13 @@ def filesystem_download
     send_data fs.contents, :filename => fs.name
   end
 
+  def adv_search_text_clear
+    if @_params[:controller] == "vm_infra"
```

Comment:
@GregP do we need to only clear search text for Infrastructure VM explorer, wondering if it will just work for all explorers without this if condition